integration/tls_syslog_server.go

```diff
@@ -35,8 +36,12 @@ func (s *TLSSyslogServer) Run() error {
 		return errors.New("failed to apped CA")
 	}
 
+	certPrefix := "fixtures/server"
+	if s.CertPrefixOverride != "" {
+		certPrefix = s.CertPrefixOverride
+	}
 	// Listen for incoming connections.
```

Comment:
Move comment above the `if` statement for style.

Reply:
Actually I moved it below cause it seems to refer to a different line entirely